scrapers/KristenBjorn.yml

```
@@ -0,0 +1,114 @@
+name: "KristenBjorn"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - kristenbjorn.com/video
+    scraper: sceneScraper
+movieByURL:
+  - action: scrapeXPath
+    url:
+      - kristenbjorn.com/web/model/store/
+    scraper: movieScraper
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - kristenbjorn.com/gay-porn-star/
+      - kristenbjorn.com/web/model/gay-porn-star/
+    scraper: performerScraper
+
+xPathScrapers:
+  sceneScraper:
+    common:
+      $performer: //div[@class="scene-item"]/a[@class="scene-box"]
+    scene:
+      Title: //article/div[1]//h2/text()
+      Details: //div[@class="med-text"]/p/text()
+      Tags:
+        Name: //a[@role="button"]/text()
+      Performers:
+        Name: $performer//h5/text()
+        URL:
+          selector: $performer/@href
+          postProcess: &appendDomain
+            - replace:
+                - regex: ^(.*)$
+                  with: http://kristenbjorn.com$1
+      Image: &imageAttr //meta[@property="og:image"]/@content
+      Studio:
+        Name:
+          fixed: Kristen Bjorn
+      Movies:
+        Name: //article/div[5]//h2/text()
```

adding a class to the mix makes it safer (less chance of a false match)
```suggestion
        Name: //article/div[@class="StoreHeaderNewContainer"][2]//h2/text()
```